selftests/unit/test_replay.py

```diff
@@ -28,12 +28,48 @@ def test_replay_map_interrupted_json(self):
                       '"PASS"}], "total": 4}')
         rep = replay.Replay()
         act = rep._create_replay_map(self.tmpdir, ["PASS"])
-        exp = [None, test.ReplaySkipTest, test.ReplaySkipTest, test.ReplaySkipTest, test.ReplaySkipTest]
+        exp = [None, test.ReplaySkipTest, test.ReplaySkipTest,
+               test.ReplaySkipTest, test.ReplaySkipTest]
         self.assertEqual(act, exp)
         act = rep._create_replay_map(self.tmpdir, ["INTERRUPTED"])
         exp = [test.ReplaySkipTest, None, None, None, None]
         self.assertEqual(act, exp)
 
+    def test_replay_map_after_crash(self):
+        """
+        Fallback to tap when json not (yet) to get executed tests
```

Comment:
Suggestion:

Tests the fallback to TAP as the source of executed tests when JSON was not generated